platforms/shared/charts/vault-k8s-mgmt/templates/serviceAccount.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: vault-auth
```

Comment:
Should be read from helm values as I mentioned above

Reply:
Done